### ISSUE TITLE
error: implicit declaration of function 'CreateJavaVM' is invalid in C99

### ISSUE THREAD

**Issue by AndreeVela** [LeeKamentsky/python-javabridge#188]
Hi, I have the following errors while trying to install javabridge on macos Catalina Version 0.15.7 :

``` 
javabridge.c:8122:36: error: implicit declaration of function 'CreateJavaVM' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
    __pyx_t_3 = __Pyx_PyInt_From_int(CreateJavaVM((&__pyx_v_self->vm), ((void **)(&__pyx_v_env)), (&__pyx_v_args))); if (unlikely(!__pyx_t_3)) __PYX_ERR(0, 651, __pyx_L1_error)
``` 

``` 
javabridge.c:9199:5: error: implicit declaration of function 'StopVM' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
      StopVM(__pyx_v_self->vm);
``` 

I'm installing from an active anaconda environment. My environment configurations are these:

``` 
% conda --version
conda 4.9.2
``` 

``` 
% python --version
Python 3.8.3
```

``` 
% /usr/libexec/java_home -V                                                 
Matching Java Virtual Machines (1):
    1.8.0_271, x86_64:	"Java SE 8"	/Library/Java/JavaVirtualMachines/jdk1.8.0_271.jdk/Contents/Home

/Library/Java/JavaVirtualMachines/jdk1.8.0_271.jdk/Contents/Home
```

Somethings that I have already tried and did not work:

- Installing using java SDK 15
- Installing outside anaconda environment.

Something that I tried and worked:
- Installing javabridge using this specific version https://github.com/LeeKamentsky/python-javabridge/issues/169#issuecomment-571585646

Although this solves the problem when directing installing javabridge, it does not allows third party libraries (i.e. python-weka-wrapper3) which depends on automatically installing javabridge being isntalled.

Thank you very much in advance guys.

**PD:**I left the complete installation command output.

``` 
ERROR: Command errored out with exit status 1:
   command: /Users/andreevela/anaconda3/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/setup.py'"'"'; __file__='"'"'/private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-wheel-9_57an9y
       cwd: /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/
  Complete output (149 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.macosx-10.9-x86_64-3.8
  creating build/lib.macosx-10.9-x86_64-3.8/javabridge
  copying javabridge/_version.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge
  copying javabridge/jutil.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge
  copying javabridge/locate.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge
  copying javabridge/__init__.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge
  copying javabridge/wrappers.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge
  copying javabridge/noseplugin.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge
  creating build/lib.macosx-10.9-x86_64-3.8/javabridge/tests
  copying javabridge/tests/__init__.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge/tests
  copying javabridge/tests/test_cpython.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge/tests
  copying javabridge/tests/test_javabridge.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge/tests
  copying javabridge/tests/test_jutil.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge/tests
  copying javabridge/tests/test_wrappers.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge/tests
  creating build/lib.macosx-10.9-x86_64-3.8/javabridge/jars
  copying javabridge/jars/rhino-1.7R4.jar -> build/lib.macosx-10.9-x86_64-3.8/javabridge/jars
  copying javabridge/jars/runnablequeue.jar -> build/lib.macosx-10.9-x86_64-3.8/javabridge/jars
  copying javabridge/jars/cpython.jar -> build/lib.macosx-10.9-x86_64-3.8/javabridge/jars
  copying javabridge/jars/test.jar -> build/lib.macosx-10.9-x86_64-3.8/javabridge/jars
  running build_ext
  javac /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/java/org/cellprofiler/runnablequeue/RunnableQueue.java
  javac /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/java/org/cellprofiler/javabridge/test/RealRect.java
  javac /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/java/org/cellprofiler/javabridge/CPython.java /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/java/org/cellprofiler/javabridge/CPythonInvocationHandler.java
  Note: /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/java/org/cellprofiler/javabridge/CPythonInvocationHandler.java uses unchecked or unsafe operations.
  Note: Recompile with -Xlint:unchecked for details.
  building 'javabridge._javabridge' extension
  creating build/temp.macosx-10.9-x86_64-3.8
  gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/Users/andreevela/anaconda3/include -arch x86_64 -I/Users/andreevela/anaconda3/include -arch x86_64 -I/Library/Java/JavaVirtualMachines/jdk1.8.0_271.jdk/Contents/Home/include -I/Library/Java/JavaVirtualMachines/jdk1.8.0_271.jdk/Contents/Home/include/darwin -I/Users/andreevela/anaconda3/lib/python3.8/site-packages/numpy/core/include -I/Users/andreevela/anaconda3/include/python3.8 -c _javabridge.c -o build/temp.macosx-10.9-x86_64-3.8/_javabridge.o
  In file included from _javabridge.c:598:
  In file included from /Users/andreevela/anaconda3/lib/python3.8/site-packages/numpy/core/include/numpy/arrayobject.h:4:
  In file included from /Users/andreevela/anaconda3/lib/python3.8/site-packages/numpy/core/include/numpy/ndarrayobject.h:12:
  In file included from /Users/andreevela/anaconda3/lib/python3.8/site-packages/numpy/core/include/numpy/ndarraytypes.h:1832:
  /Users/andreevela/anaconda3/lib/python3.8/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: "Using deprecated NumPy API, disable it with "          "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-W#warnings]
  #warning "Using deprecated NumPy API, disable it with " \
   ^
  In file included from _javabridge.c:601:
  ./mac_javabridge_utils.h:47:25: warning: this function declaration is not a prototype [-Wstrict-prototypes]
  JNIEXPORT void MacStopVM();
                          ^
                           void
  ./mac_javabridge_utils.h:55:30: warning: this function declaration is not a prototype [-Wstrict-prototypes]
  JNIEXPORT void MacRunLoopInit();
                               ^
                                void
  ./mac_javabridge_utils.h:63:29: warning: this function declaration is not a prototype [-Wstrict-prototypes]
  JNIEXPORT void MacRunLoopRun();
                              ^
                               void
  ./mac_javabridge_utils.h:71:31: warning: this function declaration is not a prototype [-Wstrict-prototypes]
  JNIEXPORT void MacRunLoopReset();
                                ^
                                 void
  ./mac_javabridge_utils.h:79:30: warning: this function declaration is not a prototype [-Wstrict-prototypes]
  JNIEXPORT void MacStopRunLoop();
                               ^
                                void
  ./mac_javabridge_utils.h:88:30: warning: this function declaration is not a prototype [-Wstrict-prototypes]
  JNIEXPORT int MacIsMainThread();
                               ^
                                void
  ./mac_javabridge_utils.h:103:20: warning: this function declaration is not a prototype [-Wstrict-prototypes]
  void MacRunLoopStop();
                     ^
                      void
  _javabridge.c:8122:36: error: implicit declaration of function 'CreateJavaVM' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
    __pyx_t_3 = __Pyx_PyInt_From_int(CreateJavaVM((&__pyx_v_self->vm), ((void **)(&__pyx_v_env)), (&__pyx_v_args))); if (unlikely(!__pyx_t_3)) __PYX_ERR(0, 651, __pyx_L1_error)
                                     ^
  _javabridge.c:8122:36: note: did you mean 'JNI_CreateJavaVM'?
  /Library/Java/JavaVirtualMachines/jdk1.8.0_271.jdk/Contents/Home/include/jni.h:1938:1: note: 'JNI_CreateJavaVM' declared here
  JNI_CreateJavaVM(JavaVM **pvm, void **penv, void *args);
  ^
  _javabridge.c:9199:5: error: implicit declaration of function 'StopVM' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
      StopVM(__pyx_v_self->vm);
      ^
  _javabridge.c:27819:3: warning: 'tp_print' is deprecated [-Wdeprecated-declarations]
    0, /*tp_print*/
    ^
  /Users/andreevela/anaconda3/include/python3.8/cpython/object.h:260:5: note: 'tp_print' has been explicitly marked deprecated here
      Py_DEPRECATED(3.8) int (*tp_print)(PyObject *, FILE *, int);
      ^
  /Users/andreevela/anaconda3/include/python3.8/pyport.h:515:54: note: expanded from macro 'Py_DEPRECATED'
  #define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                       ^
  _javabridge.c:27919:3: warning: 'tp_print' is deprecated [-Wdeprecated-declarations]
    0, /*tp_print*/
    ^
  /Users/andreevela/anaconda3/include/python3.8/cpython/object.h:260:5: note: 'tp_print' has been explicitly marked deprecated here
      Py_DEPRECATED(3.8) int (*tp_print)(PyObject *, FILE *, int);
      ^
  /Users/andreevela/anaconda3/include/python3.8/pyport.h:515:54: note: expanded from macro 'Py_DEPRECATED'
  #define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                       ^
  _javabridge.c:28050:3: warning: 'tp_print' is deprecated [-Wdeprecated-declarations]
    0, /*tp_print*/
    ^
  /Users/andreevela/anaconda3/include/python3.8/cpython/object.h:260:5: note: 'tp_print' has been explicitly marked deprecated here
      Py_DEPRECATED(3.8) int (*tp_print)(PyObject *, FILE *, int);
      ^
  /Users/andreevela/anaconda3/include/python3.8/pyport.h:515:54: note: expanded from macro 'Py_DEPRECATED'
  #define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                       ^
  _javabridge.c:28181:3: warning: 'tp_print' is deprecated [-Wdeprecated-declarations]
    0, /*tp_print*/
    ^
  /Users/andreevela/anaconda3/include/python3.8/cpython/object.h:260:5: note: 'tp_print' has been explicitly marked deprecated here
      Py_DEPRECATED(3.8) int (*tp_print)(PyObject *, FILE *, int);
      ^
  /Users/andreevela/anaconda3/include/python3.8/pyport.h:515:54: note: expanded from macro 'Py_DEPRECATED'
  #define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                       ^
  _javabridge.c:28284:3: warning: 'tp_print' is deprecated [-Wdeprecated-declarations]
    0, /*tp_print*/
    ^
  /Users/andreevela/anaconda3/include/python3.8/cpython/object.h:260:5: note: 'tp_print' has been explicitly marked deprecated here
      Py_DEPRECATED(3.8) int (*tp_print)(PyObject *, FILE *, int);
      ^
  /Users/andreevela/anaconda3/include/python3.8/pyport.h:515:54: note: expanded from macro 'Py_DEPRECATED'
  #define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                       ^
  _javabridge.c:28464:3: warning: 'tp_print' is deprecated [-Wdeprecated-declarations]
    0, /*tp_print*/
    ^
  /Users/andreevela/anaconda3/include/python3.8/cpython/object.h:260:5: note: 'tp_print' has been explicitly marked deprecated here
      Py_DEPRECATED(3.8) int (*tp_print)(PyObject *, FILE *, int);
      ^
  /Users/andreevela/anaconda3/include/python3.8/pyport.h:515:54: note: expanded from macro 'Py_DEPRECATED'
  #define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                       ^
  _javabridge.c:29747:3: warning: incompatible pointer to integer conversion returning 'void *' from a function with result type 'int' [-Wint-conversion]
    import_array();
    ^~~~~~~~~~~~~~
  /Users/andreevela/anaconda3/lib/python3.8/site-packages/numpy/core/include/numpy/__multiarray_api.h:1546:151: note: expanded from macro 'import_array'
  #define import_array() {if (_import_array() < 0) {PyErr_Print(); PyErr_SetString(PyExc_ImportError, "numpy.core.multiarray failed to import"); return NUMPY_IMPORT_ARRAY_RETVAL; } }
                                                                                                                                                        ^~~~~~~~~~~~~~~~~~~~~~~~~
  /Users/andreevela/anaconda3/lib/python3.8/site-packages/numpy/core/include/numpy/__multiarray_api.h:1541:35: note: expanded from macro 'NUMPY_IMPORT_ARRAY_RETVAL'
  #define NUMPY_IMPORT_ARRAY_RETVAL NULL
                                    ^~~~
  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include/stddef.h:89:16: note: expanded from macro 'NULL'
  #  define NULL ((void*)0)
                 ^~~~~~~~~~
  _javabridge.c:33284:26: warning: code will never be executed [-Wunreachable-code]
                  module = PyImport_ImportModuleLevelObject(
                           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  16 warnings and 2 errors generated.
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for javabridge
  Running setup.py clean for javabridge
Failed to build javabridge
Installing collected packages: javabridge, python-weka-wrapper3
  Attempting uninstall: javabridge
    Found existing installation: javabridge 0.0.0
    Uninstalling javabridge-0.0.0:
      Successfully uninstalled javabridge-0.0.0
    Running setup.py install for javabridge ... error
    ERROR: Command errored out with exit status 1:
     command: /Users/andreevela/anaconda3/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/setup.py'"'"'; __file__='"'"'/private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-record-s1ihxwf8/install-record.txt --single-version-externally-managed --compile --install-headers /Users/andreevela/anaconda3/include/python3.8/javabridge
         cwd: /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/
    Complete output (149 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.macosx-10.9-x86_64-3.8
    creating build/lib.macosx-10.9-x86_64-3.8/javabridge
    copying javabridge/_version.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge
    copying javabridge/jutil.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge
    copying javabridge/locate.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge
    copying javabridge/__init__.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge
    copying javabridge/wrappers.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge
    copying javabridge/noseplugin.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge
    creating build/lib.macosx-10.9-x86_64-3.8/javabridge/tests
    copying javabridge/tests/__init__.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge/tests
    copying javabridge/tests/test_cpython.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge/tests
    copying javabridge/tests/test_javabridge.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge/tests
    copying javabridge/tests/test_jutil.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge/tests
    copying javabridge/tests/test_wrappers.py -> build/lib.macosx-10.9-x86_64-3.8/javabridge/tests
    creating build/lib.macosx-10.9-x86_64-3.8/javabridge/jars
    copying javabridge/jars/rhino-1.7R4.jar -> build/lib.macosx-10.9-x86_64-3.8/javabridge/jars
    copying javabridge/jars/runnablequeue.jar -> build/lib.macosx-10.9-x86_64-3.8/javabridge/jars
    copying javabridge/jars/cpython.jar -> build/lib.macosx-10.9-x86_64-3.8/javabridge/jars
    copying javabridge/jars/test.jar -> build/lib.macosx-10.9-x86_64-3.8/javabridge/jars
    running build_ext
    javac /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/java/org/cellprofiler/runnablequeue/RunnableQueue.java
    javac /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/java/org/cellprofiler/javabridge/test/RealRect.java
    javac /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/java/org/cellprofiler/javabridge/CPython.java /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/java/org/cellprofiler/javabridge/CPythonInvocationHandler.java
    Note: /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/java/org/cellprofiler/javabridge/CPythonInvocationHandler.java uses unchecked or unsafe operations.
    Note: Recompile with -Xlint:unchecked for details.
    building 'javabridge._javabridge' extension
    creating build/temp.macosx-10.9-x86_64-3.8
    gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/Users/andreevela/anaconda3/include -arch x86_64 -I/Users/andreevela/anaconda3/include -arch x86_64 -I/Library/Java/JavaVirtualMachines/jdk1.8.0_271.jdk/Contents/Home/include -I/Library/Java/JavaVirtualMachines/jdk1.8.0_271.jdk/Contents/Home/include/darwin -I/Users/andreevela/anaconda3/lib/python3.8/site-packages/numpy/core/include -I/Users/andreevela/anaconda3/include/python3.8 -c _javabridge.c -o build/temp.macosx-10.9-x86_64-3.8/_javabridge.o
    In file included from _javabridge.c:598:
    In file included from /Users/andreevela/anaconda3/lib/python3.8/site-packages/numpy/core/include/numpy/arrayobject.h:4:
    In file included from /Users/andreevela/anaconda3/lib/python3.8/site-packages/numpy/core/include/numpy/ndarrayobject.h:12:
    In file included from /Users/andreevela/anaconda3/lib/python3.8/site-packages/numpy/core/include/numpy/ndarraytypes.h:1832:
    /Users/andreevela/anaconda3/lib/python3.8/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: "Using deprecated NumPy API, disable it with "          "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-W#warnings]
    #warning "Using deprecated NumPy API, disable it with " \
     ^
    In file included from _javabridge.c:601:
    ./mac_javabridge_utils.h:47:25: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    JNIEXPORT void MacStopVM();
                            ^
                             void
    ./mac_javabridge_utils.h:55:30: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    JNIEXPORT void MacRunLoopInit();
                                 ^
                                  void
    ./mac_javabridge_utils.h:63:29: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    JNIEXPORT void MacRunLoopRun();
                                ^
                                 void
    ./mac_javabridge_utils.h:71:31: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    JNIEXPORT void MacRunLoopReset();
                                  ^
                                   void
    ./mac_javabridge_utils.h:79:30: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    JNIEXPORT void MacStopRunLoop();
                                 ^
                                  void
    ./mac_javabridge_utils.h:88:30: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    JNIEXPORT int MacIsMainThread();
                                 ^
                                  void
    ./mac_javabridge_utils.h:103:20: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    void MacRunLoopStop();
                       ^
                        void
    _javabridge.c:8122:36: error: implicit declaration of function 'CreateJavaVM' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
      __pyx_t_3 = __Pyx_PyInt_From_int(CreateJavaVM((&__pyx_v_self->vm), ((void **)(&__pyx_v_env)), (&__pyx_v_args))); if (unlikely(!__pyx_t_3)) __PYX_ERR(0, 651, __pyx_L1_error)
                                       ^
    _javabridge.c:8122:36: note: did you mean 'JNI_CreateJavaVM'?
    /Library/Java/JavaVirtualMachines/jdk1.8.0_271.jdk/Contents/Home/include/jni.h:1938:1: note: 'JNI_CreateJavaVM' declared here
    JNI_CreateJavaVM(JavaVM **pvm, void **penv, void *args);
    ^
    _javabridge.c:9199:5: error: implicit declaration of function 'StopVM' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        StopVM(__pyx_v_self->vm);
        ^
    _javabridge.c:27819:3: warning: 'tp_print' is deprecated [-Wdeprecated-declarations]
      0, /*tp_print*/
      ^
    /Users/andreevela/anaconda3/include/python3.8/cpython/object.h:260:5: note: 'tp_print' has been explicitly marked deprecated here
        Py_DEPRECATED(3.8) int (*tp_print)(PyObject *, FILE *, int);
        ^
    /Users/andreevela/anaconda3/include/python3.8/pyport.h:515:54: note: expanded from macro 'Py_DEPRECATED'
    #define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                         ^
    _javabridge.c:27919:3: warning: 'tp_print' is deprecated [-Wdeprecated-declarations]
      0, /*tp_print*/
      ^
    /Users/andreevela/anaconda3/include/python3.8/cpython/object.h:260:5: note: 'tp_print' has been explicitly marked deprecated here
        Py_DEPRECATED(3.8) int (*tp_print)(PyObject *, FILE *, int);
        ^
    /Users/andreevela/anaconda3/include/python3.8/pyport.h:515:54: note: expanded from macro 'Py_DEPRECATED'
    #define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                         ^
    _javabridge.c:28050:3: warning: 'tp_print' is deprecated [-Wdeprecated-declarations]
      0, /*tp_print*/
      ^
    /Users/andreevela/anaconda3/include/python3.8/cpython/object.h:260:5: note: 'tp_print' has been explicitly marked deprecated here
        Py_DEPRECATED(3.8) int (*tp_print)(PyObject *, FILE *, int);
        ^
    /Users/andreevela/anaconda3/include/python3.8/pyport.h:515:54: note: expanded from macro 'Py_DEPRECATED'
    #define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                         ^
    _javabridge.c:28181:3: warning: 'tp_print' is deprecated [-Wdeprecated-declarations]
      0, /*tp_print*/
      ^
    /Users/andreevela/anaconda3/include/python3.8/cpython/object.h:260:5: note: 'tp_print' has been explicitly marked deprecated here
        Py_DEPRECATED(3.8) int (*tp_print)(PyObject *, FILE *, int);
        ^
    /Users/andreevela/anaconda3/include/python3.8/pyport.h:515:54: note: expanded from macro 'Py_DEPRECATED'
    #define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                         ^
    _javabridge.c:28284:3: warning: 'tp_print' is deprecated [-Wdeprecated-declarations]
      0, /*tp_print*/
      ^
    /Users/andreevela/anaconda3/include/python3.8/cpython/object.h:260:5: note: 'tp_print' has been explicitly marked deprecated here
        Py_DEPRECATED(3.8) int (*tp_print)(PyObject *, FILE *, int);
        ^
    /Users/andreevela/anaconda3/include/python3.8/pyport.h:515:54: note: expanded from macro 'Py_DEPRECATED'
    #define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                         ^
    _javabridge.c:28464:3: warning: 'tp_print' is deprecated [-Wdeprecated-declarations]
      0, /*tp_print*/
      ^
    /Users/andreevela/anaconda3/include/python3.8/cpython/object.h:260:5: note: 'tp_print' has been explicitly marked deprecated here
        Py_DEPRECATED(3.8) int (*tp_print)(PyObject *, FILE *, int);
        ^
    /Users/andreevela/anaconda3/include/python3.8/pyport.h:515:54: note: expanded from macro 'Py_DEPRECATED'
    #define Py_DEPRECATED(VERSION_UNUSED) __attribute__((__deprecated__))
                                                         ^
    _javabridge.c:29747:3: warning: incompatible pointer to integer conversion returning 'void *' from a function with result type 'int' [-Wint-conversion]
      import_array();
      ^~~~~~~~~~~~~~
    /Users/andreevela/anaconda3/lib/python3.8/site-packages/numpy/core/include/numpy/__multiarray_api.h:1546:151: note: expanded from macro 'import_array'
    #define import_array() {if (_import_array() < 0) {PyErr_Print(); PyErr_SetString(PyExc_ImportError, "numpy.core.multiarray failed to import"); return NUMPY_IMPORT_ARRAY_RETVAL; } }
  

 /Users/andreevela/anaconda3/lib/python3.8/site-packages/numpy/core/include/numpy/__multiarray_api.h:1541:35: note: expanded from macro 'NUMPY_IMPORT_ARRAY_RETVAL'
    #define NUMPY_IMPORT_ARRAY_RETVAL NULL
                                      ^~~~
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/12.0.0/include/stddef.h:89:16: note: expanded from macro 'NULL'
    #  define NULL ((void*)0)
                   ^~~~~~~~~~
    _javabridge.c:33284:26: warning: code will never be executed [-Wunreachable-code]
                    module = PyImport_ImportModuleLevelObject(
                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    16 warnings and 2 errors generated.
    error: command 'gcc' failed with exit status 1
    ----------------------------------------
  Rolling back uninstall of javabridge
  Moving to /Users/andreevela/anaconda3/lib/python3.8/site-packages/javabridge-0.0.0.dist-info/
   from /Users/andreevela/anaconda3/lib/python3.8/site-packages/~avabridge-0.0.0.dist-info
  Moving to /Users/andreevela/anaconda3/lib/python3.8/site-packages/javabridge/
   from /Users/andreevela/anaconda3/lib/python3.8/site-packages/~avabridge
ERROR: Command errored out with exit status 1: /Users/andreevela/anaconda3/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/setup.py'"'"'; __file__='"'"'/private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-install-1ececbcb/javabridge/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/gb/zc61x4qs1ns3z0cbs5w5rcx40000gn/T/pip-record-s1ihxwf8/install-record.txt --single-version-externally-managed --compile --install-headers /Users/andreevela/anaconda3/include/python3.8/javabridge Check the logs for full command output.
```

**Comment by danielsiwiec** [comment]
I'm getting the same error. Python 3.7.9, MacOs Big Sur

**Comment by LeeKamentsky** [comment]
Hi all, I think this is a duplicate of #183. This was fixed by 00cc093. Can you try installing from Github master?

pip install https://github.com/LeeKamentsky/python-javabridge/archive/master.zip#egg=python-javabridge

Please feel free to reopen if this did not work.
